titus-common/src/main/java/com/netflix/titus/common/runtime/internal/DefaultTitusRuntime.java

```diff
@@ -62,7 +62,7 @@
     private static final long INITIAL_RETRY_DELAY_MS = 10;
     private static final long MAX_RETRY_DELAY_MS = 10_000;
 
-    private static final int LOCAL_SCHEDULER_LOOP_INTERVAL_MS = 100;
+    private static final Duration LOCAL_SCHEDULER_LOOP_INTERVAL_MS = Duration.ofMillis(100);
```

Comment:
minor nit, remove the `_MS` suffix, since the unit is now captured by the `Duration` type